changelog/unreleased/issues/changes.md

```diff
@@ -0,0 +1 @@
+- In case an app provides custom `Router` implementation to Nav SDK, the Nav SDK doesn't trigger `NavigationRouteAlternativesObserver` with online alternatives when current route is offline.
```

Comment:
Is it really so? I thought it was more like "it's triggered, but much later". The alternatives mechanism is still working, right? we just won't trigger an alternative request as soon as the network is back.

Reply:
@averkhaturau, can you please clarify consequences of using custom router for offline-online switch?

Reply:
NN knows which router is online only if it creates the routers itself. Providing a custom router makes this information unavailable, we don't know how to create an online router anymore. That's why the back online feature will not be available with a custom router